### PR TITLE
docs: add note about static files being watched by default

### PR DESCRIPTION
**Pull request by anshumanv**
- [ ] This is a **bugfix**
- [ ] This is a **feature**
- [ ] This is a **code refactor**
- [ ] This is a **test update**
- [x] This is a **docs update**
- [ ] This is a **metadata update**

### For Bugs and Features; did you add new tests?

NA

### Motivation / Use-Case
add note

### Breaking Changes
no

### CODE REVIEW

File: migration-v4.md [hunk 1]
```diff
@@ -325,6 +325,7 @@ module.exports = {
 ```
 
 - Default value of the `static` option is `path.resolve(process.cwd(), 'public')` directory and enabled by default.
+- Watching of static files is enabled by default.
```
Comment (snitin315) [inline migration-v4.md:328]:
```suggestion
-  `static.watch` is enabled by default.
```

Comment (anshumanv) [reply migration-v4.md:328]: nice

Comment (snitin315) [reply migration-v4.md]: I think we have an extra space at the beginning now, can you confirm?

Comment (anshumanv) [reply migration-v4.md]: nah it looks fine

**Review by alexander-akait — DISMISSED** [review]

**Review by snitin315 — APPROVED** [review]
Thanks!

**Pull request merged**